pytest.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
+addopts = --strict-markers -s -v
+markers =
+    unit
+    integration
```

Comment:
Following the model that Badre mentioned should we go by rather with 'resource' and 'task' markers?

Reply:
IMO we should follow the "path of explicitness": unit, functional, integration marks are friendlier for newcomers (like me, hehehe)